x-pack/plugins/stack_alerts/server/alert_types/es_query/alert_type.test.ts

```diff
@@ -194,6 +195,7 @@ describe('alertType', () => {
       },
     ]);
     alertServices.scopedClusterClient.asCurrentUser.search.mockResolvedValueOnce(
+      // @ts-expect-error not compatible response type
```

Comment:
I don't know how critical is this annotations, but I think our team should work on fixing the code to remove it... I will open the proper issue.